examples/webgl2_multisampled_renderbuffers.html

```diff
@@ -172,7 +188,11 @@
 
 				const halfWidth = container.offsetWidth / 2;
 
-				group.rotation.y += clock.getDelta() * 0.1;
+				if ( params.animate ) {
+
+					group.rotation.y += 0.002;
```

Comment:
Remove clock so a restarted animation continues where it left off -- and does not jump.